README.md

```diff
@@ -17,9 +27,33 @@ Requirements:
 
 - Mamba: [Installation guide](https://mamba.readthedocs.io/en/latest/installation.html) (suggested Mambaforge).
 
-Install the orchestrator virtual environment.
+To install the `cwltool` and activate the enviroment to run the workflow:
+
+```bash
+mamba create --name cwltool_test cwltool -c conda-forge
```

Comment:
I would still generate the workflow engine env from some config file, rather than inline. To abstract from CWL engine: when deploying on different resource providers we may have different engines available, thus having a specific config file for each provider. Example: 

- CERN: `mamba env create -p ./.venv --file cwl-environment-cern.yml`
- FZJ: `mamba env create -p ./.venv --file cwl-environment-fzj.yml`

This will allow us to keep track of all this knowledge

Reply:
changed